cmake-modules/CodeCoverage.cmake

```diff
@@ -124,6 +125,22 @@ if(NOT GCOV_PATH)
     message(FATAL_ERROR "gcov not found! Aborting...")
 endif() # NOT GCOV_PATH
 
+####### C++ specific. Commenting from original or it fails for our gcc build for C99
```

Comment:
Remove the commented code

Reply:
I recommended keeping it because it makes seeing the diff between our version and the source, easier, which helps validate and keep things in sync. Currently, we have no other difference other than this.

Up to you.

Reply:
Avoiding commented out code is ideal.  Commented code tends to bit rot quickly.

As for the diff, Either way it shows up as diff.  In one view all the lines are removed and in the other they are all commented.

Reply:
removed it